misc/vtk/src/processors/vtkreader.cpp

```diff
@@ -102,7 +102,10 @@ void VTKReader::process() {
 
 VTKReader::VTKFileType VTKReader::determineFileType(const std::string& fileName) const {
     std::string line{};
-    std::ifstream infile(fileName);
+    std::ifstream infile = filesystem::ifstream(fileName);
+    if (!infile.is_open()) {
+        throw inviwo::Exception("Could not open file.", IVW_CONTEXT);
```

Comment:
Probably want to state the file path in the error message as well

Reply:
Yes, could use `fmt::format("Could not open file '{}'", fileName)` here.